pkg/scraper/scraper.go

```diff
@@ -29,10 +28,10 @@ func getDocument(url string) (*goquery.Document, error) {
 
 		defer res.Body.Close()
 
-		if res.StatusCode != 200 {
+		if res.StatusCode != http.StatusOK {
 			errorMessage := fmt.Sprintf("status code error: %d %s", res.StatusCode, res.Status)
 
-			log.Fatal(errorMessage)
+			fmt.Println(errorMessage)
```

Comment:
print error for line 41 and 48 as well

Reply:
Just for your info, you can use `log.Print` alternatively, the difference is that it includes a timestamp. Which is useful for tracking an issue in production.

We will probably change to use zerolog's `log.Error().Msg("some error")` as that will include log level, timestamp and optionally objects, data, etc.